src/libraries/System.Linq/src/System/Linq/OrderBy.cs

```diff
@@ -140,14 +140,37 @@ public static partial class Enumerable
 
         /// <summary>Gets whether the results of an unstable sort will be observably the same as a stable sort.</summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal static bool TypeIsImplicitlyStable<T>() =>
-            typeof(T) == typeof(sbyte) || typeof(T) == typeof(byte) ||
-            typeof(T) == typeof(int) || typeof(T) == typeof(uint) ||
-            typeof(T) == typeof(short) || typeof(T) == typeof(ushort) ||
-            typeof(T) == typeof(long) || typeof(T) == typeof(ulong) ||
-            typeof(T) == typeof(Int128) || typeof(T) == typeof(UInt128) ||
-            typeof(T) == typeof(nint) || typeof(T) == typeof(nuint) ||
-            typeof(T) == typeof(bool) || typeof(T) == typeof(char);
+        internal static bool TypeIsImplicitlyStable<T>()
+        {
+            // Check for integral primitive types that compare equally iff they have the same bit pattern.
+            // (bool is included because, even though technically it can have 256 different values, anything
+            // other than 0/1 is only producable using unsafe code.)
```

Comment:
```suggestion
            // other than 0/1 is only produceable using unsafe code.)
```
Or producible (TIL myself)

Reply:
I meant to say apparently it can be producible or produceable. the first one was new to me. anyway. now it's fine